src/components/LinkChooser/LinkChooser.tsx

```diff
@@ -48,13 +41,15 @@ export const LinkChooser: FC<LinkChooserProps> = ({
 }) => {
     const [{ context, matches, value }, send, service] = useMachine(
         linkChooserMachine.withContext({
+            guidelineNodes: [],
```

Comment:
What about having those in `searchResults` (it currently contains all searches or templates searches)